### PR TITLE
[WIP] update my slide

### PR DESCRIPTION
**Pull request by anthonylalba**
Summarize the changes

## Ask for reviews
@mattarzadeh @dustinzuch  can you review me?

## Create a task list
- [ ] Task to be completed

cc: @sharonmathes  @mattarzadeh

### CODE REVIEW

File: _posts/0000-01-15-anthonylalba.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Tony was here
```
Comment (dustinzuch) [inline _posts/0000-01-15-anthonylalba.md]: @anthonylalba you need to just go away.

File: _posts/0000-01-15-anthonylalba.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (dustinzuch) [inline _posts/0000-01-15-anthonylalba.md]: Garbage title

**Review by dustinzuch — COMMENTED** [review]
@anthonylalba Quit while you're NOT ahead.

File: _posts/0000-01-15-anthonylalba.md (hunk 1 above)
Comment (anthonylalba) [reply _posts/0000-01-15-anthonylalba.md]: thanks a lot really

File: _posts/0000-01-15-anthonylalba.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to our slide dick!"
```
Comment (anthonylalba) [inline _posts/0000-01-15-anthonylalba.md:3]: @dustinzuch  do you like my title now???

**Review by dustinzuch — APPROVED** [review]
GOOD!

**Pull request merged**